src/utils/icons.tsx

```diff
@@ -0,0 +1,13 @@
+import LocalDiningIcon from '@mui/icons-material/LocalDining';
+import LocalHospitalIcon from '@mui/icons-material/LocalHospital';
+import MasksIcon from '@mui/icons-material/Masks';
+import ChildFriendlyIcon from '@mui/icons-material/ChildFriendly';
+import { ReactNode } from 'react';
+
+// SVG icons for each event role
+export const roleIcons: { [id: string]: ReactNode } = {
+  Food: <LocalDiningIcon sx={{ color: '#F5A81C' }} />,
+  Medical: <LocalHospitalIcon sx={{ color: 'red' }} />,
+  Dental: <MasksIcon sx={{ color: 'lightblue' }} />,
```

Comment:
take a look at [this](https://github.com/hack4impact-utk/compassion-ministries/pull/121/files#diff-00a3a399ceb9cee803e9a3a05f2d7c296e7a9ade7798aedcf23b2ba769fcfd6dR25-R36). This is how @Frank-D-stein implemented the icons, and will ensure that no matter the order of the roles in the database that they will appear in a consistent order.